README.md

```diff
@@ -55,7 +55,7 @@ Please keep the following in mind:
 
 - All the tests for Dart exercises can be run from the top level of the repo with ... Please run this command before submitting your PR.
```

Comment:
I suggest adding the command for running all tests.

Reply:
Gotcha